tracer/src/Datadog.Trace/Debugger/Expressions/ProbeProcessor.cs

```diff
@@ -387,33 +387,41 @@ private void CheckSpanDecoration(DebuggerSnapshotCreator snapshotCreator, ref bo
                 switch (ProbeInfo.TargetSpan)
                 {
                     case TargetSpan.Root:
-                        Tracer.Instance.ScopeManager.Active.Root.Span.SetTag(decoration.TagName, decoration.Value);
-                        Tracer.Instance.ScopeManager.Active.Root.Span.SetTag(probeIdTag, ProbeInfo.ProbeId);
-                        if (decoration.Errors?.Length > 0)
+                        var rootSpan = Tracer.Instance.InternalActiveScope.Root.Span;
+                        if (rootSpan != null)
                         {
-                            Tracer.Instance.ScopeManager.Active.Root.Span.SetTag(evaluationErrorTag, string.Join(";", decoration.Errors));
-                        }
-                        else if (Tracer.Instance.ScopeManager.Active.Span.GetTag(evaluationErrorTag) != null)
-                        {
-                            Tracer.Instance.ScopeManager.Active.Root.Span.SetTag(evaluationErrorTag, null);
-                        }
+                            rootSpan.SetTag(decoration.TagName, decoration.Value);
+                            rootSpan.SetTag(probeIdTag, ProbeInfo.ProbeId);
+                            if (decoration.Errors?.Length > 0)
+                            {
+                                rootSpan.SetTag(evaluationErrorTag, string.Join(";", decoration.Errors));
+                            }
+                            else if (rootSpan.GetTag(evaluationErrorTag) != null)
+                            {
+                                rootSpan.SetTag(evaluationErrorTag, null);
+                            }
 
-                        attachedTags = true;
+                            attachedTags = true;
+                        }
 
                         break;
                     case TargetSpan.Active:
-                        Tracer.Instance.ScopeManager.Active.Span.SetTag(decoration.TagName, decoration.Value);
-                        Tracer.Instance.ScopeManager.Active.Span.SetTag(probeIdTag, ProbeInfo.ProbeId);
-                        if (decoration.Errors?.Length > 0)
+                        var activeScope = Tracer.Instance.InternalActiveScope.Span;
```

Comment:
nit: rename to `activeSpan`

Reply:
Fixed in 4f65984ac05f007e668b761591939ca4d934f150